public/i18n/de.json

```diff
@@ -150,7 +150,9 @@
         "BALLOT_MESSAGES_NOT_SUPPORTED": "Umfragen werden in Threema Web derzeit nicht untersützt.",
         "UNKNOWN_MESSAGE_TYPE": "Unbekannter Nachrichtentyp",
         "NICKNAME": "Nickname",
-        "THREEMA_WORK_CONTACT": "Threema Work Nutzer"
+        "THREEMA_WORK_CONTACT": "Threema Work Nutzer",
+        "DELETE_THREAD": "Chat löschen",
+        "DELETE_THREAD_MESSAGE": "{count, plural, one {Möchten Sie wirklich diesen Chat löschen? Die Nachrichten können nicht wiederhergestellt werden.} other {Möchten Sie wirklich # Chats löschen? Die Nachrichten können nicht wiederhergestellt werden.}}"
```

Comment:
Do you plan to support deleting multiple chats at once? So far we only use count 1. If not, I'd remove the plural forms.

Reply:
yes is planned...